src/components/FloatingButton.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+import { motion } from "framer-motion";
+import { MdOutlineViewInAr } from "react-icons/md";
+import "./floating-button.css";
+
+interface FloatingButtonProps {
+  onClick: () => void;
+  label: string;
+}
+
+export default function FloatingButton({
```

Comment:
please make the button a little bit more subtle
- remove the shadown waves
- make button a little bit wider